website/docs/result_to_dict.md

```diff
@@ -0,0 +1,106 @@
+---
+id: result_to_dict
+title: Result to Dictionary
+sidebar_label: Result to Dictionary
+---
+## result_to_dict() Function
+Will format any value a function returns.
+For e.g., will return tuple as a tuple, dict as a dict
+
+Datatypes supported:
+
+ - Point (Geographical point)
+ - Varchar
+ - ENUM
+ - Integer
+ - Test
+ - JSON will be converted to `dict`

```

Comment:
`will be converted to `dict`` to `(converted to dict)`
also for the rest same format